test/helpers/enableSnapshot.js

```diff
@@ -0,0 +1,12 @@
+import chai from 'chai'
+import chaiJestSnapshot from 'chai-jest-snapshot'
+
+chai.use(chaiJestSnapshot)
+
+before(function () {
```

Comment:
I dislike hiding any magic with test helpers, and before hooks definitely fall into that territory for me.

However, this is a pretty good extraction, and I don't see any reason not to keep it. I think we should rename files, so it is clear from far away where the magic happens. This file might be `enableSnapshotUsingBeforeHooks.js` while `test/components/elements/MenuLinkTest.js` becomes `test/components/elements/MenuLinkSnapshotTest.js`. I'm not sure tho… what do you think?

Reply:
Renaming to `enableSnapshotUsingBeforeHook.js`: agreed 
Renaming snapshot tests... need more information: Business logic tests can also live in that file because we're testing the whole component... in this case there's no need for business logic tests on that component, but something more complex like `NavMenu` would still need business logic tests for `mapStateToProps`, etc.